### DISCUSSION TITLE
Epochs - good practices

### DISCUSSION THREAD

**Discussion by patalanov** [voicepaw/so-vits-svc-fork#855] [General]
How many epochs are in general recommended for a dataset with around 500 samples? 
How many epochs for 1000 samples? 
How many epochs for 1500 samples?

**Comment by nikich340** [comment]
Don't forget to check losses in tensorboard (with ~0.98 smoothing) and preview audios.
Usually it takes 500-1000 epochs for good results (you can try more even if losses stagnates, but always make D and G models backups).
I don't think there will be big difference between 500 or 1500 samples, but if you can, surely use more. I use 500-1000 samples usually.
